### PR TITLE
Add config for the dev environment via application-dev.yaml

### PR DESCRIPTION
**Pull request by nelsestu**
## What was the problem?
Dev environment is still using localhost as the BIP URL

Associated tickets or Slack threads:
- #2298, #2297

## How does this fix it?[^1]
Update the bipClaimBaseUrl property so dev points to a real dev env for BIP.

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

**Review by chengjie8 — APPROVED** [review]

File: svc-bip-api/src/main/resources/application-dev.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+bip:
+  claimBaseUrl: "https://claims-dev.dev.bip.va.gov"
+  applicationId: "${BIP_APPLICATION_ID:VRO}"
+  stationId: "${BIP_STATION_ID:456}"
+  env: dev
```
Comment (tejans24) [inline svc-bip-api/src/main/resources/application-dev.yml:6]: Needs a new line here?

**Review by tejans24 — APPROVED** [review]
LGTM with a nit fix

File: svc-bip-api/src/main/resources/application-dev.yml (hunk 1 above)
Comment (msnwatson) [inline svc-bip-api/src/main/resources/application-dev.yml:6]: Nit newline at eof

**Review by msnwatson — APPROVED** [review]

File: svc-bip-api/src/main/resources/application-dev.yml (hunk 1 above)
Comment (chengjie8) [reply svc-bip-api/src/main/resources/application-dev.yml:6]: `./gradlew spotlessApply`

Comment (nelsestu) [reply svc-bip-api/src/main/resources/application-dev.yml]: Newline added

**Pull request merged**